### PR TITLE
Descriptions for Fields

### PR DESCRIPTION
**Pull request by ODORA0**

### CODE REVIEW

**Comment by ODORA0** [comment]
@ssmusoke Added descriptions for the fields as requested by David, please have a look

File: reference/recency-surveillance.v1.json [hunk 1]
```diff
@@ -177,25 +191,30 @@
...
                       "last_hiv_test_result": {
+                        "description": "Test results the last time tested",
                         "type": "string",
                         "enum": [
                           "HIV+",
                           "HIV-"
                         ]
                       },
                       "last_hiv_test_date": {
+                        "description": "Must be YYYY-MM-DD format",
```
Comment (mugumedavid) [inline reference/recency-surveillance.v1.json]: Its small yyyy capital MM and small dd. The casing is important. Same for the other date fields as well.

Comment (ODORA0) [reply reference/recency-surveillance.v1.json]: @mugumedavid Done

File: reference/recency-surveillance.v1.json [hunk 2]
```diff
@@ -293,10 +312,12 @@
...
                       "accompanied_by_other": {
+                        "description": "Only filled if accompanied_by is other",
                         "type": "string",
                         "default": "text"
                       },
                       "accompanied_by": {
+                        "description": "If the client is a child below 12 years",
                         "type": "string",
                         "enum": [
                           "GRANDPARENT",
```
Comment (mugumedavid) [inline reference/recency-surveillance.v1.json]: @ODORA0  there's also these capital cases which shouldn't be. Unless its an abbreviation like PITC.

Comment (ODORA0) [reply reference/recency-surveillance.v1.json]: @mugumedavid That's OK, I'll change all of them. The time I made these changes I got them directly as they were in the EMR

Comment (ODORA0) [reply reference/recency-surveillance.v1.json]: @mugumedavid Done

File: reference/recency-surveillance.v1.json [hunk 3]
```diff
@@ -59,150 +59,155 @@
...
                         "type": "string",
-                        "enum": [
-                          "No",
-                          "Yes"
-                        ]
+                        "default": "text"
                       },
-                      "referred_to_tb_services": {
+                      "visit_date": {
+                        "description": "Must be YYYY-MM-DD format",
```
Comment (ssmusoke) [inline reference/recency-surveillance.v1.json]: @ODORA0 update the description to match the default format and also an example date to illustrate

File: reference/recency-surveillance.v1.json [hunk 4]
```diff
@@ -59,150 +59,155 @@
...
                     ],
                     "properties": {
+                      "description": "DHIS2 organisation unit uuid for this facility",
                       "dhis2_orgunit_uuid": {
                         "type": "string",
                         "default": "text"
                       },
-                      "referred_location": {
+                      "encounter_uuid": {
+                        "description": "Encounter universal unique identifier",
```
Comment (ssmusoke) [inline reference/recency-surveillance.v1.json]: @ODORA0 updated description, Encounter universal unique identifier usually a uuid for example c0d8ed6d-386a-4a1b-b9e7-af246fee0ca3

File: reference/recency-surveillance.v1.json [hunk 5]
```diff
@@ -59,150 +59,155 @@
...
                         "enum": [
-                          "No",
-                          "Yes"
+                          "M",
+                          "F"
                         ]
                       },
-                      "recent_hiv_test_result": {
+                      "date_of_birth": {
+                        "description": "Must be YYYY-MM-DD format",
```
Comment (ssmusoke) [inline reference/recency-surveillance.v1.json]: @ODORA0 update the description to match the default format and also an example date to illustrate

Also add the date when the client was born

File: reference/recency-surveillance.v1.json [hunk 6]
```diff
@@ -223,139 +228,164 @@
...
-                          "Self Initiative",
-                          "PMTCT",
-                          "Others"
+                          "No",
+                          "Yes"
                         ]
                       },
-                      "community_testing_entry_point_other": {
+                      "last_hiv_test_date": {
+                        "description": "Must be YYYY-MM-DD format",
```
Comment (ssmusoke) [inline reference/recency-surveillance.v1.json]: @ODORA0 update the description to match the default format and also an example date to illustrate

File: reference/recency-surveillance.v1.json (hunk 4 above)
Comment (ODORA0) [reply reference/recency-surveillance.v1.json]: @ssmusoke Done

File: reference/recency-surveillance.v1.json (hunk 3 above)
Comment (ODORA0) [reply reference/recency-surveillance.v1.json]: @ssmusoke Done

File: reference/recency-surveillance.v1.json (hunk 6 above)
Comment (ODORA0) [reply reference/recency-surveillance.v1.json]: @ssmusoke Done

File: reference/recency-surveillance.v1.json (hunk 5 above)
Comment (ODORA0) [reply reference/recency-surveillance.v1.json]: @ssmusoke Done, but what did you mean by the date when the client was born?

File: reference/recency-surveillance.v1.json [hunk 7]
```diff
@@ -59,150 +59,155 @@
...
                         "type": "string",
-                        "enum": [
-                          "No",
-                          "Yes"
-                        ]
+                        "default": "yyyy-MM-dd ",
+                        "format": "date"
                       },
-                      "presumtive_tb": {
+                      "Parish": {
```
Comment (mugumedavid) [inline reference/recency-surveillance.v1.json:83]: This is lower case `parish`

**Review by mugumedavid — APPROVED** [review]
This is good now. My review is complete.

**Pull request merged**